backend/src/main/java/com/jobwait/security/linkedin/LinkedInUserProfile.java

```diff
@@ -0,0 +1,7 @@
+package com.jobwait.security.linkedin;
+
+public class LinkedInUserProfile {
+    public String sub; // UUID
+    public String email;
+    public Boolean email_verified;
```

Comment:
Do we care about these two email fields?

Reply:
No, but I kept them in regardless because it may be useful later and I think there was some peculiarity with not passing these fields giving me some weird error.